From 06871bd4866fb8ea1e1bc58f5374b0069c072ee8 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Sun, 11 Aug 2024 20:35:18 -0700
Subject: [PATCH] Build with C++17

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 4b0c268..ec9423a 100644
--- a/Makefile
+++ b/Makefile
@@ -101,10 +101,10 @@ obj/libco.o: libco/libco.c
 
 ifeq ($(platform),ios-arm64)
    c := $(CC) -Wno-error=implicit-function-declaration -std=gnu99
-   cpp := $(CXX) -std=c++11 -stdlib=libc++ #-std=gnu++0x
+   cpp := $(CXX) -std=c++17 -stdlib=libc++ #-std=gnu++0x
 else
    c := $(CC) -std=gnu99
-   cpp := $(CXX) #-std=gnu++0x
+   cpp := $(CXX) -std=c++17
 endif
 
 flags += $(opt) -fomit-frame-pointer -fno-tree-vectorize -I. $(fpic)
-- 
2.34.1

